.github/CODEOWNERS

```diff
@@ -37,8 +37,10 @@ src/applications/static-pages @department-of-veterans-affairs/vsa-public-website
 src/applications/find-forms @department-of-veterans-affairs/vsa-public-websites-frontend @department-of-veterans-affairs/vsa-facilities-frontend @department-of-veterans-affairs/vsa-caregiver-frontend
 src/applications/yellow-ribbon @department-of-veterans-affairs/vsa-public-websites-frontend @department-of-veterans-affairs/vsa-facilities-frontend @department-of-veterans-affairs/vsa-caregiver-frontend
 
-# Facility Locator
+# Facility Locator and VAMC pages
 src/applications/facility-locator @department-of-veterans-affairs/vsa-facilities-frontend @department-of-veterans-affairs/vsa-public-websites-frontend
+src/applications/static-pages/facilities @department-of-veterans-affairs/vsa-facilities-frontend
+src/site/layouts @department-of-veterans-affairs/vsa-facilities-frontend
```

Comment:
Is there a specific layout that the facilities team needs access to? Since there are a lot of layouts it may be better to specify the paths to the layouts needed.

Reply:
I have been wanting to arrange our VAMC pages into one folder but my attempt with moving the liquid templates around breaks the build. So we either need some help on how to re-arrange these templates or we need some time to go one by one cc @mpelzsherman @ncksllvn

Reply:
My personal preference would be to move the VAMC-related templates into one folder, and fix the build issues.

This would require assistance from the platform and CMS teams, but it would help us tremendously.

Until that happens, I think the most pragmatic solution is to give facilities team ownership of all CMS templates, so we can be notified of any changes that might break areas of VA.gov that we are responsible for.

Reply:
@theodur, ^ Do you think we can get full access for now? maybe we could add @department-of-veterans-affairs/vsa-public-websites-frontend as a secondary owner?

Reply:
@brangi Yeah that should be fine for now. I just approved the PR.

Reply:
I think we should have set the frontend review group as an additional code owner of `src/site/layouts`, because now VSA facilities will have to approve all layout changes (even those unrelated to VAMC products) such as https://github.com/department-of-veterans-affairs/vets-website/pull/14038

Reply:
Or yeah my team `vsa-public-websites-frontend ` is a good one to add as well